apps/system/js/audio_channel_manager.js

```diff
@@ -0,0 +1,267 @@
+/* global BaseModule */
+'use strict';
+
+(function() {
+  var PLAY_CURRENT_AND_NEW_CHANNEL = 0;
+  var PAUSE_CURRENT_CHANNEL = 1;
+  var PAUSE_THEN_RESUME_CURRENT_CHANNEL = 2;
+  var VOLUME_DOWN_CURRENT_CHANNEL = 3;
+  var VOLUME_DOWN_NEW_CHANNEL = 4;
+  var PAUSE_THEN_VIBRATE_CURRENT_CHANNEL = 5;
+  var VIBRATE_NEW_CHANNEL = 6;
```

Comment:
@dominickuo,
We don't need to expose these constants. I don't know any other module will need it in the future.
We only expose these two method `register`, `unregister` now.
How do you think?